themes/default/content/templates/container-service/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Container Templates
+layout: overview
+description: Pulumi program templates are the fastest way to deploy [containerized applications](/containers) on [AWS](/aws), [Azure](/azure), or [Google Cloud Platform](/gcp). Templates come with predefined [infrastructure as code](/what-is/what-is-infrastructure-as-code) so you can get started instantly.
```

Comment:
Similarly to the others, let's pull the links out of this section and put them into the body of the page.

Reply:
```suggestion
description: Pulumi program templates are the fastest way to deploy [container services](/containers) on [AWS](/aws), [Azure](/azure), or [Google Cloud Platform](/gcp). Templates come with predefined [infrastructure as code](/what-is/what-is-infrastructure-as-code) so you can get started instantly.
```

i would say container services rather than the containerized applications